packages/ui/src/organisms/OutOfStock/stories/OutOfStock.mdx

```diff
@@ -0,0 +1,35 @@
+import { Canvas, Props, Story, ArgsTable } from '@storybook/addon-docs'
+
+import OutOfStock from '../OutOfStock'
+
+# OutOfStock
+
+<Canvas>
+  <Story id="organisms-outofstock--out-of-stock" />
+</Canvas>
+
+The `OutOfStock` uses the [Compound Component](https://kentcdodds.com/blog/compound-components-with-react-hooks) pattern, its components are:
+
+- `OutOfStock`: the form wrapper of the out of stock component with title and message;
```

Comment:
I think it also good to list `<OutOfStock.Title>` and `<OutOfStock.Message>` here.